src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -645,10 +650,12 @@ private void UpdateCacheIfNeeded(long position, ITensorValueGetter[] srcTensorGe
                     Runner runner = new Runner(_parent.Session);
 
                     // Feed inputs to the graph.
+                    List<Tensor> inputTensors = new List<Tensor>();
                     for (int i = 0; i < _parent.Inputs.Length; i++)
                     {
                         var tensor = srcTensorGetters[i].GetTensor();
                         runner.AddInput(_parent.Inputs[i], tensor);
+                        inputTensors.Add(tensor);
```

Comment:
Instead of adding these to another list - would it be better to just get the inputs from the `runner` and dispose of them?